app/helpers/mountain_view/styleguide_helper.rb

```diff
@@ -1,5 +1,13 @@
 module MountainView
   module StyleguideHelper
+    def extra_pages
+      MountainView.configuration.extra_pages
+    end
+
+    def prettify_word(word)
+      word.to_s.split('_').map(&:capitalize).join(' ')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.